test_package/example.cpp

```diff
@@ -0,0 +1,6 @@
+#include <iostream>
+#include "hello.h"
```

Comment:
I doubt that this can actually be built via conan. There's no "hello.h", and furthermore I'd definitely say that such a test package should be written in plain C89, not C++

Reply:
I've updated this, but haven't pushed yet, since it's not a working build, and I don't want to have 1k commits, each with a minor fix.